spoon-dataflow/README.md

```diff
@@ -32,26 +32,7 @@ Check out test directory for more examples.
 In general, spoon-dataflow is capable to evaluate expressions statically, perform symbolic execution, handle control flow of a program, and so on. It also features a proper memory model, so it nicely deals with reference aliasing.
 
 ## Build and run
-In order to build spoon-dataflow you need JDK 8 or newer. Also, you have to download and install Z3.
-
-### Windows:
-1. Download z3-4.8.4 here: https://github.com/Z3Prover/z3/releases
-2. Add bin directory to your PATH environment variable
-3. Run `gradlew build`
-
-Note: Visual C++ 2015 Redistributable may be required.
-
-### Linux:
-1. Download z3-4.8.4 here https://github.com/Z3Prover/z3/releases
-2. Add bin directory to your LD_LIBRARY_PATH: `export LD_LIBRARY_PATH=/path/to/z3/bin`
-3. Run `./gradlew build`
-
-### macOS:
-1. Download z3-4.8.4 here https://github.com/Z3Prover/z3/releases
-2. Add bin directory to your DYLD_LIBRARY_PATH: `export DYLD_LIBRARY_PATH=/path/to/z3/bin`
-3. Run `./gradlew build`
-
-Note that you may still need to set up environment variables or java.lang.path in your IDE.
+In order to build spoon-dataflow you need JDK 8 or newer.
```

Comment:
Spoon in current versions needs java 11. Are you sure with this statement?

Reply:
That was there before and works for sufficiently old versions of spoon-core. So maybe we should leave it as is?

Reply:
We should replace the `+` with a real version. As currently, spoon data flow isn't published, at least I am not aware of, any user needs to build it locally. The `+` gets translated to the newest resolvable version, which is hopefully a current spoon version. And this version needs 11.

Reply:
I updated the message a while ago to state that you need java 11 due to spoon-core :)

Reply:
Oh sorry github was showing the old content. What a nice UI. LGTM we can merge